grammar.js

```diff
@@ -1140,8 +1140,7 @@ module.exports = grammar({
     _backtick_identifier: $ => /`[^\r\n`]+`/,
 
     _uni_character_literal: $ => seq(
-      "\\",
-      "u",
+      "\\u",
```

Comment:
looks identical, what is the difference?

Reply:
The separate sequence `seq("\\", "u")` wasn't parsing unicode correctly in the example, but combining the two into `\\u` resulted in no errors 😱. I think it has something to do with how tree-sitter processes sequences of literals, but I'm not completely sure. 

It does increase parsing though :D